### DISCUSSION TITLE
Intro

### DISCUSSION THREAD

**Discussion by Priyanshu-bhati23** [Cyfrin/foundry-full-course-cu#3051] [General]
Hello guys! i am priyanshu an enginnerig grad.looking forward to interact with all of you

**Comment by EngrPips** [comment]
Hello @Priyanshu-bhati23, Welcome aboard, and we are glad to have you.

**Reply by Priyanshu-bhati23** [reply]
Excited to be a part of this community

**Comment by Conradgabe** [comment]
Hi guys, i'm Gabriel, i am diving deeper into web3 and security, looking forward to interacting with everyone

**Reply by EngrPips** [reply]
Welcome aboard @Conradgabe, We wish you the best.
